app/src/main/java/io/gnosis/safe/ui/transactions/paging/TransactionPagingSource.kt

```diff
@@ -3,20 +3,23 @@ package io.gnosis.safe.ui.transactions.paging
 import androidx.paging.PagingSource
 import io.gnosis.data.models.transaction.TxListEntry
 import io.gnosis.data.repositories.TransactionRepository
+import io.gnosis.safe.ui.transactions.TransactionListFragment
 import pm.gnosis.model.Solidity
 import timber.log.Timber
 
 class TransactionPagingSource(
     private val safe: Solidity.Address,
-    private val txRepo: TransactionRepository
+    private val txRepo: TransactionRepository,
+    private val type: TransactionListFragment.Type
```

Comment:
this Type enum is used outside of fragment now. I propose to put it into TransactionPagingSource. This way it will read itself quite nicely: TransactionPagingSource.Type (HISTORY | QUEUE)